### PR TITLE
MOB-3061 - Add Adyen payment sample code for logged in users

### PR DESCRIPTION
**Pull request by jsm20**
## JIRA
[MOB-3061](https://karhoo.atlassian.net/browse/MOB-3061)

## DESCRIPTION
Add sample code to show integration of booking using Adyen payments for logged in users 

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [x] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
N/a

### CODE REVIEW

File: network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+                //Handle error
+            }
+        }
+    }
+
+    private fun threeDSecureNonce(
+        paymentsNonce: String
+    ) {
+        showLoading()
+//        bindCardDetails(
```
Comment (mohadian) [inline network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt:150]: Shall we remove the commented code?

File: network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+import com.karhoo.sdk.api.network.request.Passengers
+import com.karhoo.sdk.api.network.request.TripBooking
+import com.karhoo.sdk.api.network.response.Resource
+import kotlinx.android.synthetic.main.fragment_trip_booking.*
+import org.json.JSONObject
+import java.util.*
+
+class AdyenTripBookingFragment : BaseFragment(), UserManager.OnUserPaymentChangedListener {
+
+    private var locale: String = "GB"
```
Comment (mohadian) [inline network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt:50]: Could add a constant to the companion object as DEFAULT_LOCALE

Comment (jsm20) [reply network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt]: Done

File: network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt (hunk 1 above)
Comment (jsm20) [reply network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt]: Updated to bind the card details

**Review by mohadian — APPROVED** [review]

**Pull request merged**